DeUrgenta/Features/CreateGroup/View/CreateGroupView.swift

```diff
@@ -0,0 +1,59 @@
+import SwiftUI
+
+struct CreateGroupView: View {
+    var body: some View {
+        VStack(alignment: .leading) {
+            HStack {
+                Text(AppStrings.CreateGroup.headerTitle.localized())
+                    .font(.custom("IBMPlexSans-SemiBold", size: 24))
+                    .padding(.leading, 20)
+                    .padding(.top, 10)
+                Spacer()
+            }
+            VStack {
+                Text(AppStrings.CreateGroup.firstParagraph.localized())
+                    .font(.custom("IBMPlexSans-Regular", size: 16))
+                    .foregroundColor(.darkText)
+                Text(AppStrings.CreateGroup.secondParagraph.localized())
+                    .padding(.top, 15)
+                    .font(.custom("IBMPlexSans-Regular", size: 16))
+                    .foregroundColor(.darkText)
+            }
+            .padding(.leading, 20)
+            .padding(.trailing, 20)
+            .padding(.top, 5)
+            .lineSpacing(10)
+            Spacer()
+            VStack {
+                Button(action: {
+                    // TODO: Implement this action
```

Comment:
Let's also implement these methods - probably we need some redirects via coordinators to other screens - it is easy to redirect to an empty screen. In this way, other contributors can quickly start coding the following screens.

Reply:
Ok, I will do that.